src/utils/StripedLoader.js

```diff
@@ -3,8 +3,8 @@ import { observer } from 'mobx-react'
 import { loading } from '../styles/generic'
 
 let StripedLoader = (Component, style = {}) =>
-  observer(props => (
-    <div style={{ ...style, ...(props.loading && loading) }}>
+  observer(({ isLoading, ...props }) => (
```

Comment:
loading should still be passed to the component, because some components might customize their rendering if they know they’re loading. Component should just be mindful not to blindly pass all props to a native Dom element